### DISCUSSION TITLE
Does SEB support on windows 11 ?

### DISCUSSION THREAD

**Discussion by loctran7777** [SafeExamBrowser/seb-win-refactoring#223] [Q&A]
I just upgraded to win 11 and now I'm confused because my school is about to apply the exam on SEB. I looked it up, it seems that seb only supports win 10. Will there be SEB on win 11?

**Comment by dbuechel** [comment]
According to our preliminary tests, SEB works fine on Windows 11. We will update our website once our testing has performed a thorough examination of SEB 3.x under Windows 11.

**Reply by Peryasio** [reply]
so does it work now?

**Reply by Peryasio** [reply]
actually yes

**Reply by dbuechel** [reply]
We might get around to do an official release testing for version 3.3.2, but most certainly for version 3.4. As already stated, according to preliminary tests and also due to the fact that Windows 11 mainly brings user interface changes, SEB should work fine on Windows 11.

**Comment by nvarghese-rm** [comment]
@dbuechel  - Our clients did some testing on SEB Win v3.3.1 with Windows 11 and found it did not work. Then they tested v3.3.2 with Windows 11 and it worked. 

Have you observed his behaviour with 3.3.1 and Windows 11?
Has there been some work from your end that you have actually done for Windows 11 that has now addressed this ?

**Reply by dbuechel** [reply]
No, on our side the preliminary tests worked fine with all recent builds of SEB 3.x. We'd need to see the exact details which caused SEB to not work on Windows 11, but I couldn't really explain that behavior as we haven't addressed anything specifically for Windows 11.
